lib/croutons/breadcrumb_trail.rb

```diff
@@ -15,16 +15,16 @@ def initialize(template_identifer, objects = {})
       build_breadcrumbs
     end
 
-    def method_missing(name, *args)
+    def method_missing(name, *args, &block)
```

Comment:
Agreed that the Rails URL helpers don't take a block as an argument, but I'm not sure what the purpose of capturing the block is if we don't do anything with it.